### PR TITLE
NAS-110580 / 21.06 / Fields are not properly aligned in application creation form

### PR DESCRIPTION
**Pull request by AlexKarpov98**

### CODE REVIEW

**Comment by denysbutenko** [comment]
image
`Apps -> Advanced Settings`

**Comment by AlexKarpov98** [comment]
@undsoft Fixed.

File: src/app/pages/common/entity/entity-form/entity-form.component.html [hunk 1]
```diff
@@ -4,31 +4,31 @@
 </p>
 <mat-card class="form-card">
   <!-- <div *ngIf="conf.title" class="mat-toolbar mat-card-toolbar" fxLayoutAlign="space-between center">
```
Comment (denysbutenko) [inline src/app/pages/common/entity/entity-form/entity-form.component.html:6]: do we need to keep this?

**Review by denysbutenko — APPROVED** [review]
image
looks good

**Pull request merged**